htdocs/reports/reports_testhistory.php

```diff
@@ -789,11 +789,14 @@ function change_orientation() {
 	&nbsp;&nbsp;&nbsp;&nbsp;
 			<input type='button' onClick="javascript:print_content('report_content');" value='<?php echo LangUtil::$generalTerms['CMD_PRINT']; ?>'></input>
 			<div id="dialog" title="Basic dialog"></div>
+			
 	</td>
 	<td>
 		<table class='no border'>
 	<tr valign='top'>
-		
+	<td>
+		<input type="checkbox" id="page-break" name="page-break" value="page-break"><label for="vehicle1"><?php echo LangUtil::$generalTerms['ONE_TEST_PER_PAGE']; ?></label>

```

Comment:
What is the `for="vehicle1"` bit referring to?

Reply:
I copied an example from w3schools while prototyping and forgot to remove it.